requirements.txt

```diff
@@ -1,7 +1,7 @@
 ovos-plugin-manager>=0.0.1
-ovos-utils~=0.0.27a3
+ovos-utils>=0.0.27
```

Comment:
This could probably be `~=0.0.27` to prevent updating to an incompatible release (i.e. 1.0.0) in the future